docs/devtools.md

```diff
@@ -136,25 +117,42 @@ Additional Reads:
 
 ## PostgreSQL Tips:
 
-Connect to Postgres
+**Connect to Postgre**
 ```
 PGPASSWORD=postgres psql postgres -U postgres -h localhost -p 15432
 ```
 
```

Comment:
Another handy way of managing connections is to use a [connection service file](https://www.postgresql.org/docs/current/libpq-pgservice.html).

Example `~/.pg_service.conf`:
```ini
[koku_db]
host=localhost
dbname=postgres
port=5432
user=postgres
password=postgres
```

Then you can set the `PGSERVICE` variable and connect.
```
PGSERVICE=koku_db psql
```

Reply:
@samdoran I like this idea, you want to push a commit up to the branch suggesting this alternative too?

Reply:
Added!